mesa/visualization/modules/BarChartVisualization.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+"""
+Pie Chart Module
+============
+
+Module for drawing live-updating bar charts using d3.js
+
+"""
+import json
+from mesa.visualization.ModularVisualization import VisualizationElement
+
+
+class BarChartModule(VisualizationElement):
+    """ Each bar chart can either visualize model-level or agent-level fields from a datcollector
+        with a bar chart.
+
+    Attributes:
+        scope: wheter to visualize agent-level or model-level fields
+        fields: A List of Dictionaries containing information about each field to be charted,
+                including the name of the datacollector field and the desired color of the
+                cooresponding bar.
+                Ex: [{"Label":"<your field name>", "Color":"<your desired color in hex>"}]
+        sorting: Wheter to sort ascending, descending, or neither when charting agent fields
+        sort_by: The agent field to sort by
+        canvas_height, canvas_width: The width and height to draw the chart on the page, in pixels.
+                                    Default to 800 x 400
+        data_collector_name: Name of the DataCollector object in the model to retrieve data from.
+
+    """
+
+    package_includes = ["d3.min.js", "BarChartModule.js"]
+
+    def __init__(self, fields, scope="model", sorting="none",
+                sort_by="none", canvas_height=400,
+                canvas_width=800, data_collector_name="datacollector"):
+        """
```

Comment:
This seems to be the same documentation as in the class description above, but with slightly different wording?

Reply:
So this is one of those places where I followed the example from the pervious chart example, but where the previous example didn't really make sense to me. I agree that the repetition is odd, but I assumed it had something to do with the way the automatic documentation generation worked. 

I tried to somewhat differentiate what was in the class comment (things that you should know *about* the class) and the initializer comment (things you should know about *how* to use the class), but I would some guidance on if this was the right move or really just about how the documentation on something like this should go. (This is my first semi-big experience writing code/documentation for other people to read aside from my CS class TA)